spec/features/user_can_like_comments_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
```

Comment:
Nice. We could also consider adding some spec helpers for creating posts and comments because these tests are starting to look chunky.